.github/workflows/test.yml

```
@@ -20,7 +20,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
+        node-version: [18.x, 20.x]
```

Once v22 is release we switch from 21 to 22
```suggestion
        node-version: [18.x, 20.x, 21.x]
```